python-project/test/test_jp.py

```diff
@@ -105,10 +105,12 @@ def test_rudra(index):
 
 def ocr_test_case(test_case: OCRTest, index: int) -> None:
     name = f'{test_case.base_name}-{index + 1:02}'
-    input_path = f'{png_dir(test_case)}/{name}.png'
+    input_png_path = f'''{dir_mkr(test_case, 'png')}/{name}.png'''
+    input_txt_path = f'''{dir_mkr(test_case, 'txt')}/{name}.txt'''
     output_path = mktemp(suffix='.png')
 
-    image = PIL.Image.open(input_path)
+    image = PIL.Image.open(input_png_path)
+    expected_text = Path(input_txt_path).read_text()
```

Comment:
`Path` can take a string argument, so reading the contents of a file becomes a one-liner.